src/core/ddsi/src/ddsi_raweth.c

```diff
@@ -397,7 +526,7 @@ int ddsi_raweth_init (struct ddsi_domaingv *gv)
   fact->gv = gv;
   fact->m_free_fn = ddsi_raweth_deinit;
   fact->m_typename = "raweth";
-  fact->m_default_spdp_address = "raweth/ff:ff:ff:ff:ff:ff";
+  fact->m_default_spdp_address = "raweth/01:00:5e:7f:00:01";
```

Comment:
(I presume this comes from the spec? Then of course I am happy with the change. I used a broadcast originally because, well, why not for a quick hack.)